scheduler/snippets/job_test.py

```diff
@@ -14,21 +14,23 @@
 
 import os
 
+from _pytest.capture import CaptureFixture
```

Comment:
Don't need if you also return a value (even if it's a string that was also printed).